sass/_component_versionpicker_fix.scss

```diff
@@ -15,6 +15,7 @@ These overrides are necessary to make the version picker work with the Sphinx Wa
 
   &.shift-up .rst-other-versions {
     display: block;
+    max-height: calc(100vh - 50px);
```

Comment:
I think ideally this should be

```scss
  &.shift-up {
    max-height: calc(100vh - 50px);

    .rst-other-versions {
      display: block;
    }
  }
```

And the `max-height: calc(100vh - 50px);` from a few lines above should be removed.

Reply:
ok thanks, I was not sure if the non-shift-up variant was ever used but this makes sense